models/gpg_key_commit_verification.go

```diff
@@ -86,6 +89,67 @@ func ParseCommitsWithSignature(oldCommits []*UserCommit, repository *Repository)
 	return newCommits
 }
 
+// ParseCommitWithSignature check if signature is good against keystore.
+func ParseCommitWithSSHSignature(c *git.Commit, committer *User) *CommitVerification {
```

Comment:
Just to note that there is a lot of duplicate code with the `ParseCommitWithSignature` function(where this function is called from).